### ISSUE TITLE
security exploit

### ISSUE THREAD

**Issue by martinshkreli** [flashbots/mev-boost#652]
lost $25m in 12 seconds please push patch

**Comment by Helperofme** [comment]
Maybe you should charge more for RX meds and try again.

**Comment by 7etsuo** [comment]
ouch.

**Comment by martinshkreli** [comment]
> Maybe you should charge more for RX meds and try again.

i don't think that is related to this codebase ser, please remove your comment as it is not helpful

**Comment by laserjobs** [comment]
You may be able to recover upto 25K if you file a bug report
https://github.com/flashbots/mev-boost/blob/develop/SECURITY.md

**Comment by grantbelford** [comment]
https://www.justice.gov/opa/media/1351996/dl

#26. Relay exploit.
